k8s/namespaces/em/em-npa/prod.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: em-npa
+spec:
+  values:
+    java:
```

Comment:
Don't u need to set these? https://github.com/hmcts/cnp-flux-config/pull/7647/files#diff-75adbbb47bc22a3aafdb1984aef03e8f917b283a709ff161b6c36e174d92f7deL23-L24

Reply:
We are ok with the default values at the moment.